frontend/src/ContextProviders/PlayersContextProvider.tsx

```diff
@@ -9,22 +9,32 @@ const equals = (player1: Player, player2: Player) =>
   player1.nickname === player2.nickname;
 
 type Context = {
+  host: string;
+  setHost: (host: string) => void;
```

Comment:
We should similarly add a `me` field to this

Reply:
This is no longer required as of #68 as we can use the players `id` persisted with `use-persisted-state` matched against other player ids in the `PlayersContext`